src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToControlGeneratedAssemblyInfo.cs

```diff
@@ -370,6 +370,30 @@ public void It_includes_internals_visible_to()
             AssemblyInfo.Get(assemblyPath)["InternalsVisibleToAttribute"].Should().Be("Tests");
         }
 
+        [Fact]
```

Comment:
I'm having trouble building locally on windows and testing locally on WSL2. 

On WSL2, I'm submitting this command line:
``` csharp
~/sdk/artifacts/bin/redist/Debug/dotnet/dotnet test --filter "Microsoft.NET.Build.Tests.GivenThatWeWantToControlGeneratedAssemblyInfo.It_includes_requires_preview_features*"
```

But that fails with the message that It_includes_requires_preview_features is not found. I inspected the test dll and I do see It_includes_requires_preview_features as one of the unit tests. Anyone know what I'm doing wrong?

Reply:
I'm curious, what ended up being the issue here?